app/constants/live.js

```diff
@@ -7,4 +7,13 @@ export const BUTTONS_TYPE = {
   COPY_LINK: 'copy-link',
   LEAVE_ROOM: 'leave-room',
 };
+
 export const API_METHOD = { POST: 'POST', GET: 'GET', PATCH: 'PATCH' };
+
+export const PATCH_API_CONFIGS = {
+  method: API_METHOD.PATCH,
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  credentials: 'include',
+};
```

Comment:
why method-specific config? than we will also need separate ones for POST and GEt